magefile.go

```diff
@@ -788,6 +789,9 @@ func (Cloud) Image(ctx context.Context) {
 	variant := os.Getenv(dockerVariants)
 	defer os.Setenv(dockerVariants, variant)
 
+	fips := os.Getenv(fipsEnv)
+	defer os.Setenv(fipsEnv, fips)
```

Comment:
Why is the env set back to the original - potentially invalid - value with `defer` when also you specifically set it to `true` or `false` in line (809:818)?

Reply:
I'm being consistent with how Image handles the `SNAPSHOT` env var